config/smart_answers/rates/minimum_wage.yml

```diff
@@ -258,3 +258,20 @@
     :rate: 8.91
   :apprentice_rate: 4.30
   :accommodation_rate: 8.36
+- :start_date: 2022-04-01
```

Comment:
Do you know why we need to keep data dating back to 2016? I can't see how it's used for the two services referenced in your PR description. Is it used for other services that support checking pay entitlement dating back that far? If it is, perhaps we could add a comment at the top of this file saying so?

Reply:
OK it doesn't look like it's used anywhere else. I think we should remove the old data and cut down the tests, as it makes these uprating tasks more complicated than they need to be. As it's a bit time consuming lets make a card for that and do it as a separate task.

Reply:
I'm not sure why we keep data from that long ago but this it the case with a lot of those calculators. Removing it seems like something worth doing but since it will require a lot of changes to the tests, it's probably out of scope for this PR.
The services only use current and previous year's data, I will add a comment to reflect that.

Reply:
https://trello.com/c/MslkvTUc/319-remove-legacy-data-from-smart-answers-to-make-uprating-easier